docs/synapse/userguides/storm_ref_pivot.rstorm

```diff
@@ -382,6 +385,70 @@ by a light edge is "one way"). From a Storm syntax perspective, light edges can
 - There are no light edges (i.e., specific light edge verbs) defined in a Cortex by default. Users can create and define their own according to their needs.
 - The Storm :ref:`storm-model`, :ref:`storm-edges`, and :ref:`storm-lift-byverb` commands can be used to work with  light edges in a Cortex.
 
+.. _join-light-edge:
+
+Traverse (Walk) and Join Light Edges
+------------------------------------
+
+Much like the :ref:`walk-light-edge` operators, the traverse (walk) and join light edges operator (``-(<verb>)+>`` or ``<+(<verb>)-``) is used to traverse from a set of inbound nodes to the set of nodes they are linked to by the specified light edge(s). Unlike :ref:`walk-light-edge`, the inbound nodes are retained and combined with the results of the pivot.
+
+**Syntax:**
+
+- *Walk and Join - Single Light Edge*
+
+  *<query>* **-(** *<verb>* **)+>** ***** | *<form>*
+  
+  *<query>* **<+(** *<verb>* **)-** ***** | *<form>*
+
+- *Walk and Join - Multiple Light Edges*
+
+  *<query>* **-( (** *<verb1>* **,** *<verb2>*  [ **,** *<verb3>* ...] **) )+>** ***** | *<form>*
+  
+  *<query>* **<+( (** *<verb1>* **,** *<verb2>*  [ **,** *<verb3>* ...] **) )-** ***** | *<form>*
+
+- *Walk and Join - Any Light Edge*
+
+  *<query>* **-(** ***** **)+>** ***** | *<form>*
+  
+  *<query>* **<+(** ***** **)-** ***** | *<form>*
+
+**Examples:**
+
+*Traverse the "refs" light edge from an article and join the article with the FQDNs "referenced" by the article*
+
+.. storm-pre:: [media:news="*" +(refs)> { inet:fqdn=woot.com } ] -(refs)+> inet:fqdn
+::
+
+   <media:news> -(refs)+> inet:fqdn
+
+*Join an article with any/all the nodes referenced by the article*
+
+.. storm-pre::  media:news -(refs)+> *
+::
+
+   <media:news> -(refs)+> *
+
+*Traverse the "hasip" light edge from an IPv4 address to the CIDR block(s) the IP is part of and join the two sets together:*
```

Comment:
Looks like some of these example descriptions have a `:` at the end while others do not, probably add them to the ones missing it